_includes/app/BasicSample.java

```diff
@@ -0,0 +1,53 @@
+import java.sql.*;
+
+/*
+You can compile and run this example with a command like:
```

Comment:
I'd recommend the Apache projects' style for multi-line /* comments - with a space and asterisk and another space at start of each subsequent line.

For example, see:
https://github.com/apache/lucene-solr/blob/master/lucene/analysis/common/src/java/org/apache/lucene/analysis/charfilter/BaseCharFilter.java